clients/iota-go/iotaclient/iotaclienttest/api_transaction_builder_test.go

```diff
@@ -124,7 +124,9 @@ func TestMoveCall(t *testing.T) {
 		},
 	)
 	require.NoError(t, err)
-	queryEventsResMap := queryEventsRes.Data[0].ParsedJson.(map[string]interface{})
+	queryEventsResMap := make(map[string]interface{})
```

Comment:
Avoid allocating the map twice:
```suggestion
	var queryEventsResMap map[string]any
```